.reuse/dep5

```diff
@@ -0,0 +1,33 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ROOT-Sim
+Upstream-Contact: Andrea Piccione <piccione@diag.uniroma1.it>
```

Comment:
Change to: HPDCS Research Group <rootsim@googlegroups.com>

Reply:
Yep, I missed that! Anyway I'll change it to "HPDCS Group <rootsim@googlegroups.com>" for consistency with the other SPDX headers, if we agree.